x-pack/plugins/security_solution/cypress/e2e/exceptions/add_edit_flyout/flyout_validation.cy.ts

```diff
@@ -311,6 +320,39 @@ describe('Exceptions flyout', () => {
     closeExceptionBuilderFlyout();
   });
 
+  it('Warns users about mapping conflicts on problematic field selection', async () => {
+    // open add exception modal
+    openExceptionFlyoutFromEmptyViewerPrompt();
+
+    // find 'doc_id' field which has mapping conflicts accross different indices
+    cy.get(FIELD_INPUT).eq(0).click({ force: true });
+    cy.get(FIELD_INPUT).eq(0).type('doc_id');
+
+    // hovering field should show warning tooltip
+    cy.get(EXCEPTION_FIELD_MAPPING_CONFLICTS_ICON).eq(0).trigger('mouseover');
+    cy.get(EXCEPTION_FIELD_MAPPING_CONFLICTS_TOOLTIP).should('be.visible');
+    cy.get(EXCEPTION_FIELD_MAPPING_CONFLICTS_TOOLTIP).should(
+      'have.text',
+      'This field is defined as several types across different indices.boolean: conflicts-0002long: conflicts-0001'
+    );
+
+    // select problematic field
+    cy.get(FIELD_INPUT).eq(0).type(`{downarrow}{enter}`);
+
+    // check that we show warning after the field selection in form of accordion component
+    // underneath the field component and clicking on which reveals the extended warning message
+    // with conflicts details
+    cy.get(EXCEPTION_FIELD_MAPPING_CONFLICTS_ACCORDION_ICON).eq(0).click({ force: true });
+    cy.get(EXCEPTION_FIELD_MAPPING_CONFLICTS_DESCRIPTION)
+      .eq(0)
+      .should(
+        'have.text',
+        'This field is defined as several types across different indices.boolean: conflicts-0002long: conflicts-0001'
+      );
+
+    closeExceptionBuilderFlyout();
```

Comment:
We can move all the `closeExceptionBuilderFlyout();` method into a beforeEach hook :)

Reply:
done